Gemfile

```diff
@@ -52,3 +52,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'rubocop'
```

Comment:
I tend to leave a blank line in the bottom of every file.
You can view that when the red sign is there.

In `rubocop.yml` is there too.